src/entrypoint.S

```diff
@@ -39,22 +39,6 @@ data_init:
 	jal data_cache_hit_writeback_invalidate
 	nop
 
-	/* fill .bss with 0s */
```

Comment:
What handles this for non-cpp files now? Is bss zero-filling considered unnecessary since C doesn't guarantee any particular value in a statically allocated variable?